e2e/app/component/select-menu.ts

```diff
@@ -39,7 +45,7 @@ export default class SelectMenu {
       if (maxAttempts <= 0) {
         return null;
       }
-      return await this.page.waitFor(2000).then(clickAndCheck); // two seconds pause and retry
+      return await this._page.waitFor(2000).then(clickAndCheck); // two seconds pause and retry
```

Comment:
So would `this.page` ever have evaluated to the global `page` here? Or is it just a change in name to avoid confusing humans (which I support)?

Reply:
I'd prefer renaming the member in the foreign class on import and using the best (local) name in your own class.